src/gridtools/clang/verify.hpp

```diff
@@ -25,247 +25,293 @@
 
 namespace gridtools {
```

Comment:
I think you didn't pull? Formatting seems odd? Or at least I would not expect namespaces to create indentation

Reply:
as we discussed, after this PR I will remove the .clang-format file we have in this dir, and reformat everything here.